registries/registries.py

```diff
@@ -0,0 +1,150 @@
+# System-wide registries parsing tool
+
+import sys
+from abc import abstractmethod, ABCMeta
+import yaml
+import pytoml
+import argparse
+import json
+import logging
+import logging.handlers
+import os
+
+def log_warning(conf_file):
+    test = os.environ.get("TEST", 0)
+    if test == 0:
+        log = logging.getLogger(__name__)
+        log.setLevel(logging.DEBUG)
+        handler = logging.handlers.SysLogHandler(address = '/dev/log')
+        formatter = logging.Formatter('%(module)s.%(funcName)s: %(message)s')
+        handler.setFormatter(formatter)
+        log.addHandler(handler)
+        log.warning("{} is in YAML format and should be in TOML format. Back this file up and then "
+                    "use /usr/libexec/registries_migrator > /etc/containers/registries.conf to "
+                    "convert it to TOML.".format(conf_file))
+
+map_output = {
+    "registries.search": "--add-registry",
+    "registries.insecure":  "--insecure-registry",
+    "registries.block": "--block-registry"
+}
+
+
+def write_file(filename, data):
+    f = open(filename,"w")
```

Comment:
this could be `with open(filename, 'w') as f:` and drop the `close` statement

Reply:
fixed.